addons/hr_expense/models/account_move.py

```diff
@@ -49,6 +49,13 @@ def _compute_payment_state(self):
             move.payment_state = move._get_invoice_in_payment_state()
         super(AccountMove, self - company_paid)._compute_payment_state()
 
+    @api.depends('expense_sheet_id')
+    def _compute_invoice_payment_term_id(self):
+        super()._compute_invoice_payment_term_id()
+        for move in self:
+            if move.expense_sheet_id:
+                move.invoice_payment_term_id = False
+
```

Comment:
If you mean that we put the date_maturity equal to the accounting date, it seems to be expected for expenses

Reply:
I was under the impression that we were trying to use the payment terms of the employee contact... in which case the `invoice_payment_term_id` should apply
but, if that's not the case ... 🤷‍♂️

Reply:
> we were trying to use the payment terms of the employee contact
perhaps I should double check this

Reply:
> we were trying to use the payment terms of the employee contact

I double checked to avoid confusion and indeed the only payment term setting is on the `res.partner` form which would be used for accounting purposes, but, confirmed by the expense PO, payment terms have no purpose in the scope of expenses paid by employee

Reply:
Chatted with @Laura-labe and she reconsidered and confirmed:
The contact payment terms of the contact should be used, and the due date should change if the payment terms/ due date are changed on the bill

Reply:
Alright, thanks for checking, PR updated